test_vm/Cheffile.lock

```diff
@@ -20,7 +20,7 @@ SITE
       apt (>= 0.0.0)
       yum (>= 0.0.0)
       yum-epel (>= 0.0.0)
-    mysql (6.1.0)
+    mysql (6.1.1)
```

Comment:
If we want to update `Cheffile.lock` - I think it's better to raise it as separate PR.